PWGDQ/Core/HistogramsLibrary.cxx

```diff
@@ -831,7 +831,7 @@ void o2::aod::dqhistograms::DefineHistograms(HistogramManager* hm, const char* h
         hm->AddHistogram(histClass, "U2Q2_CentFT0C_ev1", "mass vs. centrality vs. U2Q2_event1", false, 125, 0.0, 5.0, VarManager::kMass, 9, 0.0, 90.0, VarManager::kCentFT0C, 100, -10.0, 10.0, VarManager::kU2Q2Ev1);
         hm->AddHistogram(histClass, "U2Q2_CentFT0C_ev2", "mass vs. centrality vs. U2Q2_event2", false, 125, 0.0, 5.0, VarManager::kMass, 9, 0.0, 90.0, VarManager::kCentFT0C, 100, -10.0, 10.0, VarManager::kU2Q2Ev2);
       }
-      if (subGroupStr.Contains("dimuon-polarizationHE")) {
+      if (subGroupStr.Contains("dimuon-polarization-hf")) {
```

Comment:
is there a typo? Do you mean "polarization-he" instead of "polarization-hf" ?

Reply:
Hi @lucamicheletti93, I changed to hf acronym of  **Helicity Frame**.